docs/src/components/Contributors/Contributor.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { IContributor } from ".";
+import styled, { css } from "styled-components";
+import Twitter from "@kiwicom/orbit-components/lib/icons/Twitter";
+import Link from "@kiwicom/orbit-components/lib/icons/Link";
+import { Stack } from "@kiwicom/orbit-components";
+import Dribble from "../../images/dribbble.svg";
+import data from "./data.json";
+
+const resolveSize = (size: "small" | "medium" | "large") => {
+  if (size === "small") return "60px";
+  if (size === "medium") return "100px";
+
+  return "124px";
+};
+
+const StyledImageWrapper = styled(({ url, children, ...props }) => {
+  const Component = url ? "a" : "div";
+  return (
+    <Component href={url} {...props}>
+      {children}
+    </Component>
+  );
+})`
+  width: ${({ size }) => resolveSize(size)};
+`;
+
+const StyledHeading = styled.h4`
+  color: ${({ theme }) => theme.orbit.colorHeading};
+  font-size: calc(1em + 2px);
+  font-weight: 800;
```

Comment:
It would probably be better to use the `fontWeightBold` token for this because the designs are in Circular Pro, and I'm not sure if that typeface has this font weight. We should try to [include Circular Pro](https://orbit.kiwi/foundation/typography/circular-pro/) to soon in a separate PR, but I'm not sure if it's as simple as it seems.

Reply:
not sure, if we can't include `Circular Pro`, because of licence issue